.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - run:
           pip install cairo-lang;
           cargo test -- --include-ignored
+          cargo test --no-default-features --features alloc -- --include-ignored
```

Comment:
why do you want to run tests without std ?

Reply:
Oh sorry it's an additional test not a substitute of the tests with std. Can you separate this test in another job ? Want to make clear if tests are failing if this is only with std our without std.

Reply:
done